elf/module.go

```diff
@@ -770,3 +773,14 @@ func (b *Module) CloseExt(options map[string]CloseOptions) error {
 	}
 	return nil
 }
+
+// UpdateKprobeSecName updates the name on the kprobe section object from "oldName" to "newName".
+// Note that this doesn't change the key in the mapping
+func (b *Module) UpdateKprobeSecName(oldName, newName string) error {
```

Comment:
nit: I find the terminology being used here a bit confusing. The section name can be pretty much anything, it's just a way to reference code in a ELF file. 

* The problem we're trying to solve here: this package currently infers the Kprobe name from the ELF section name, but we want to have in the same object file different Kprobe *handlers* for a given Kprobe.
* The (palliative) solution: this method let's you specify the *Kprobe name* for a given *Kprobe handler*.

Can you update the method description (maybe some variable names) to make that clear?

Reply:
What about `UpdateKprobeHandler(secName, handlerName string)` then?

Reply:
humm, so handler is *your* code, the Kprobe *name* is what you're specifying. However the package refers to handlers as kprobes, so I don't know how to eliminate the confusion lol.
at least I would try explain this clearly in the method documentation.

Reply:
I'll change the name to `UpdateKprobeNameForHandler` and explain the details in the comment.